Error while checking build/O2/fullCI for 657148ba992dfd2c9c2fcca42b175b9bc7d0c489 at 2023-04-11 12:26:
```
## sw/BUILD/QualityControl-latest/log
/sw/SOURCES/QualityControl/v1.97.1/v1.97.1/Modules/EMCAL/src/ClusterTask.cxx:467:22: error: 'class o2::emcal::ClusterFactory<o2::emcal::Cell>' has no member named 'setClustersContainer'; did you mean 'gsl::span<const o2::emcal::Cluster> o2::emcal::ClusterFactory<o2::emcal::Cell>::mClustersContainer'? (not accessible from this context)
/sw/SOURCES/QualityControl/v1.97.1/v1.97.1/Modules/EMCAL/src/ClusterTask.cxx:468:22: error: 'class o2::emcal::ClusterFactory<o2::emcal::Cell>' has no member named 'setCellsContainer'; did you mean 'setContainer'?
/sw/SOURCES/QualityControl/v1.97.1/v1.97.1/Modules/EMCAL/src/ClusterTask.cxx:469:22: error: 'class o2::emcal::ClusterFactory<o2::emcal::Cell>' has no member named 'setCellsIndicesContainer'
ninja: build stopped: cannot make progress due to previous errors.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/11139/657148ba992dfd2c9c2fcca42b175b9bc7d0c489/build_O2_fullCI/pretty.html).

--- a/Detectors/EMCAL/base/include/EMCALBase/ClusterFactory.h
+++ b/Detectors/EMCAL/base/include/EMCALBase/ClusterFactory.h
@@ -331,21 +331,6 @@ class ClusterFactory
   bool getUseWeightExotic() const { return mUseWeightExotic; }
   void setUseWeightExotic(float useWeightExotic) { mUseWeightExotic = useWeightExotic; }
 
-  void setClustersContainer(gsl::span<const o2::emcal::Cluster> clusterContainer)
-  {
-    mClustersContainer = clusterContainer;
-  }
-
-  void setCellsContainer(gsl::span<const InputType> cellContainer)
-  {
-    mInputsContainer = cellContainer;
-  }
-
-  void setCellsIndicesContainer(gsl::span<const int> indicesContainer)
-  {
-    mCellsIndices = indicesContainer;
-  }
-
   void setContainer(gsl::span<const o2::emcal::Cluster> clusterContainer, gsl::span<const InputType> cellContainer, gsl::span<const int> indicesContainer)
   {
     mClustersContainer = clusterContainer;

--- a/Detectors/EMCAL/workflow/src/AnalysisClusterSpec.cxx
+++ b/Detectors/EMCAL/workflow/src/AnalysisClusterSpec.cxx
@@ -138,9 +138,7 @@ void AnalysisClusterSpec<InputType>::run(framework::ProcessingContext& ctx)
     auto inputEvent = mEventHandler->buildEvent(iev);
 
     mClusterFactory->reset();
-    mClusterFactory->setClustersContainer(inputEvent.mClusters);
-    mClusterFactory->setCellsContainer(Inputs);
-    mClusterFactory->setCellsIndicesContainer(inputEvent.mCellIndices);
+    mClusterFactory->setContainer(inputEvent.mClusters, Inputs, inputEvent.mCellIndices);
 
     //for (const auto& analysisCluster : mClusterFactory) {
     for (int icl = 0; icl < mClusterFactory->getNumberOfClusters(); icl++) {

